recipes/jyg/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: jyg
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jyg/jyg-{{ version }}.tar.gz
+  sha256: 8f65eadc0eb79f4169f43909b7401996ba30a1d46361ddafd92668c8c76901b6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - jupyter-jyg = jyg.jygapp:main
+    - jyg = jyg.jygapp:main
+
+requirements:
+  host:
+    - flit-core >=3.7.1,<4
+    - pip
+    - python >=3.8
+  run:
+    - jinja2 >=3.1
+    - jupyter_core >=4.7.1
+    - python >=3.8
+
+test:
+  imports:
+    - jyg
+  commands:
+    - pip check
+    - jupyter-jyg --help
+    - jyg --help
+    - python -m jyg --help
+    - jupyter labextension list
+    - jupyter server extension list
+    - jupyter labextension list 1>labextensions 2>&1
+    - jupyter server extension list 1>server_extensions 2>&1
+    - grep -iE "@deathbeds/jyg.*OK" labextensions
+    - grep -iE "jyg.*OK" server_extensions
+  requires:
+    - jupyterlab
+    - m2-grep  # [win]
```

Comment:
To handle this with noarch, I think you need `__unix # [not win]` and `__win # [win]` in runtime deps, right? That will prevent an incorrect package being published for Windows until you can add the necessary conda-forge.yml [as we did here](https://github.com/conda-forge/ipykernel-feedstock/pull/127/files#diff-478786365dd93f740eb520c2faa03d7a0623273663b75472272c8ef94297bbe2R9-R12)

Reply:
i have never run into issues with _test_ dependencies having selectors... i can do some looking...